setup.py

```diff
@@ -54,7 +54,7 @@ def read(*rnames):
             "plone.app.testing[robot]>=4.2.2",
             "plone.app.robotframework",
             "plone.dexterity",
-            "Products.CMFCore",
+            "Products.CMFCore>=3.1",
```

Comment:
@jaroel can you elaborate on why this version pin is needed? One of the tests fails because of this.

Reply:
@tisto that's because we need the interface introduced in that version.
We could use a local interface that just won't have any effect.

in plone.rest.interfaces do this
```python
try:
    import Products.CMFCore.interfaces.IShouldAllowAcquiredItemPublication
except ImportError:
    class IShouldAllowAcquiredItemPublication(zope.interface.Interface):
        pass
```
and adjust configure.zcml to match that.

That way we can drop the version pin.

Reply:
I think it is fine to set a minimum version. Are there any use-cases where this could be a problem?

Reply:
I am also fine with a minimum version pin in general. I just wanted to understand why this is necessary. 

However, looking at Plone 5.2 and Plone 6.0, which are currently supported by plone.rest 4.x, I see this:

```
https://dist.plone.org/release/5.2.14/versions.cfg -> Products.CMFCore 2.7.1
https://dist.plone.org/release/6.0.0/versions.cfg -> Products.CMFCore = 2.6.0
https://dist.plone.org/release/6.0.6/versions.cfg -> Products.CMFCore = 3.0
https://dist.plone.org/release/6.0.7/versions.cfg -> Products.CMFCore 3.2
```

This means that with Plone 5.2.x and Plone from 6.0.0 to 6.0.6 you would run into version conflicts. One could argue that we just need to support the latest versions and we do not care about the rest. Though, I know that those version pins can be very annoying when they start to pop up out of nowhere.

What we need to do depends on where @mauritsvanrees wants to use plone.rest 4.x. If we only use it in Plone 6.0 (currently 3.x) and Plone 6.1, we should be fine.

Reply:
@mauritsvanrees @jensens since @jaroel already adapted the code I'd say we stick with what we have, do a release, and move on. Do you agree?

Reply:
That seems fine.